src/modules/render-day.js

```diff
@@ -0,0 +1,21 @@
+import renderList from './renderlist';
+
+function renderDay(userList) {
+  const currentDay = document.querySelector('.current-date');
+  const curWeekDate = new Date().getDay();
```

Comment:
Add a separate variable for new Date() in order not to define twice what is a date now , for example, `const now = new Date()`